### PR TITLE
Feature/stub confidential migrations

### PR DESCRIPTION
**Pull request by agduncan94**
Stubs of the confidential migrations are now included. The reason for this is that we need them to exist for db migrate to work. They are included in the general migration file, and if they can't be found then the migration won't work.

### CODE REVIEW

**Comment by agduncan94** [comment]
Seems like the files require some content at minimum. At least the databaseChangeLog tag. Looks like I'll have to check in these files after all.

File: dockstore-webservice/pom.xml [hunk 1]
```diff
@@ -605,7 +605,7 @@
 
     <build>
         <plugins>
-	    <plugin>
+	        <plugin>
```
Comment (denis-yuen) [inline dockstore-webservice/pom.xml]: Nitpicky whitespace typo probably

Comment (garyluu) [inline dockstore-webservice/pom.xml]: Indents seems weird but I guess it doesn't really matter

Comment (agduncan94) [reply dockstore-webservice/pom.xml]: I don't actually see this locally, it is just the one tab.

Comment (denis-yuen) [reply dockstore-webservice/pom.xml]: @agduncan94  Might be a tab vs spaces thing

**Pull request merged**